src/components/CreateCard/Cards.test.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import Cards from "./Cards";
+import * as rtl from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
+import {Provider} from "react-redux";
+import store from "../../state/store";
+import {BrowserRouter} from "react-router-dom";
+
+let wrapper;
+beforeEach(() => {
+  wrapper = rtl.render(
+    <Provider store={store}>
+      <BrowserRouter>
+        <Cards />
+      </BrowserRouter>
+    </Provider>
+  );
+});
+// setup the cleanup
+afterEach(rtl.cleanup);
+
+it("renders without crashing", () => {
+  expect(wrapper.container).toMatchSnapshot();
+});
```

Comment:
This test doesn't test for anything and, if I assume correctly, makes real axios calls.
Ideally you would test that it renders things you give to it. The snapshot could be empty for all we know.